### DISCUSSION TITLE
Option to remove location data from backed up photos

### DISCUSSION THREAD

**Discussion by vishnukvmd** [ente-io/ente#937] [Enhancements]
Source: https://www.reddit.com/r/enteio/comments/149unbq/suggestion/

**Comment by davidcollini** [comment]
These should be the three options people should have

1. delete location data entirely

2. restrict location data from being shared outside Ente (maybe this can be the default option)

3. don't remove or restrict sharing of location data

**Comment by t-m-w** [comment]
"How do I use Ente to share photos of my cat without also sharing my home address?" is a dilemma that deserves a solution, apart from needing to remember to turn off Location in the camera app whenever you're at home. :black_cat:

**Comment by dinobagio** [comment]
Even though I prefer a native Ente solution, there might be alternative in the meantime since the location removal is sometimes possible on device galleries

**Comment by ajsteele** [comment]
It would also be useful to allow creation of shared links that restrict location (and perhaps all) metadata. I like to have all my metadata for personal use so I can see photos on the map etc, but it would be great to be able to create a public link to share with acquaintances that doesn't give away the location of every photo.

Thanks!
